Web/starter_project_iconics/components/layout/Header.tsx

```diff
@@ -0,0 +1,118 @@
+import Image from 'next/image'
+import { useState } from 'react'
+import { MdOutlineClose, MdViewHeadline } from 'react-icons/md'
+import logo from '../../public/a2sv-logo.png'
+
+function classNames(...classes: string[]) {
+  return classes.filter(Boolean).join(' ')
+}
+
+const Header = () => {
```

Comment:
can you use a syntax like React.FC (or whatever is the proper way for next) to define components?
It clarifies what is what